liberty-archetype-webapp/src/test/resources/projects/it/archetype.properties

```diff
@@ -0,0 +1,5 @@
+groupId=test.wlp
+artifactId=ServletSample
+version=0.1-SNAPSHOT
+wlpRuntimeVersion=16.0.0.4
+wlpPluginVersion=1.3-SNAPSHOT
```

Comment:
could (should?) we be using this property in the archetype-metadata.xml file above?  If not, the two will need to be kept in sync.